content/src/fieldConfig/config.json

```diff
@@ -526,12 +526,14 @@
     "cancelButtonText": "Cancel"
   },
   "cannabisPriorityStatus": {
-    "phraseWithOnePriority": "It sounds like you're eligible as ${priorityStatusOne}",
-    "phraseWithTwoPriorities": "It sounds like you're eligible as ${priorityStatusOne} and ${priorityStatusTwo}",
-    "phraseWithThreePriorities": "It sounds like you're eligible as ${priorityStatusOne}, ${priorityStatusTwo}, and ${priorityStatusThree}"
+    "phrase1": "It sounds like you're eligible as ${type1}",
+    "phrase2": "It sounds like you're eligible as ${type1} and ${type2}",
+    "phrase3": "It sounds like you're eligible as ${type1}, ${type2}, and ${type3}"
   },
   "cannabisPriorityTypes": {
+    "minorityOrWomen": "a Diversely-Owned Business",
     "minorityWomenOrVeteran": "a Diversely-Owned Business",
+    "veteran": "a Diversely-Owned Business",
```

Comment:
we don't need these new lines (they were for the other for-loop approach)

Reply:
fixed and updated tests